src/modules/auth/components/SetPasswordSuccess/SetPasswordSuccess.js

```diff
@@ -29,7 +29,7 @@ function SetPasswordSuccess({ onClose, encryptedPhrase, headerText, contentText,
         </p>
         <DownloadJSON fileName={fileName} encryptedPhrase={encryptedPhrase} />
         <PrimaryButton className={styles.continueButton} onClick={onContinue}>
-          {buttonText || t('Continue to dashboard')}
+          {buttonText || t('Continue to account')}
```

Comment:
```suggestion
          {buttonText || t('Continue to wallet')}
```

Reply:
For now lets keep it wallet, we need to rethink `wallet vs accounts`, until then `wallet` will be our reference